secboot/encrypt_sb.go

```diff
@@ -63,31 +51,7 @@ func FormatEncryptedDevice(key EncryptionKey, label, node string) error {
 //
 // A heuristic memory cost is used.
```

Comment:
this comment should be removed as well if we are no longer using our own heuristic and instead using the heuristic of upstream secboot